cmdutil/debug/config.go

```diff
@@ -2,5 +2,8 @@ package debug
 
 // Config describes the configurable parameters for debugging.
 type Config struct {
-	Port int `env:"DEBUG_PORT,default=9999"`
+	Port                 int  `env:"DEBUG_PORT,default=9999"`
+	PprofPort            int  `env:"PPROF_DEBUG_PORT,default=9998"`
+	EnablePprofDebugging bool `env:"ENABLE_PPROF_DEBUGGING,default=false"`
+	MutexProfileFraction int  `env:"MUTEX_PROFILE_FRACTION,default=2"`
```

Comment:
We should not enable mutex profiling by default as there is a performance penalty. Could you add an option to enable it?

Reply:
That's my bad @ypaq. There is an option above to enable pprof altogether, default to false